pkg/service/record/record.go

```diff
@@ -29,28 +28,28 @@ func NewRecorder(logger *zap.Logger) Recorder {
 	}
 }
 
-func (r *recorder) CaptureTraffic(path string, proxyPort uint32, appCmd, appContainer, appNetwork string, Delay uint64, buildDelay time.Duration, ports []uint, filters *models.TestFilter, enableTele bool, passThroughHosts []models.Filters) {
+func (r *recorder) CaptureTraffic(captureTrafficConfig models.TrafficCaptureParams) {
```

Comment:
if recordOption format is followed this would be just options(captureTrafficConfig)

Reply:
Refactored